index.html

```diff
@@ -0,0 +1,150 @@
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale =1.0">
+        <title>Portfolio project</title>
+    <link rel="stylesheet" href="styles.css">
+    <script src="https://kit.fontawesome.com/d901e0cb9a.js" crossorigin="anonymous"></script>
+    <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+<style>
+body {
+    font-family: 'Poppins';
+}
+</style>
+    </head>
+    <body>
+      <nav>
+        <ul>
+          <li><a class="logolink" href="">My Logo</a></li>
+          <li style="float: right;"><a href=""><img src="hammenu.png" alt="ham" style="width: 45px;height: 45px;"></a></li>
+        </ul>
+      </nav>
+        <header>
+            <div class="textdiv"><h1>I’m Alexandra <br>
+            Glad to see you!</h1>
+            <p>I’m a software developer! I can help you build a product , feature or website Look <br> through 
+              some of my work and experience! If you like what you see and have a project <br>
+              you need coded, don’t hestiate to contact me.</p>
+              <p style="color: #7F8CFF;">LET’S CONNECT</p>
+            <section><i class="fa-brands fa-twitter"></i>
+              <i class="fa-brands fa-linkedin-in"></i>
+              <i class="fa-solid fa-m"></i>
+              <i class="fa-brands fa-github"></i>
+              <i class="fa-regular fa-hand-peace"></i></section>
+            </div>
```

Comment:
- [ ] [Optional] I would suggest you please use ul tag  to group them and li to list them and each icon should have a link (`a` tag) for better semantic HTML

Reply:
Hi Roshan, Thanks for your valuable review regarding the Project and thanks for approving . I also have 2nd Project to submit I will try to incorporate those changes in the Second Project.